### DISCUSSION TITLE
[up-flashes] status

### DISCUSSION THREAD

**Discussion by alexandremjacques** [unpoly/unpoly#679] [Q&A] [answered]
Hi.

I'm woking on a new app and would like to use the `up-flashes` feature. 

Since it's tagged as Experimental, I wonder if its going to be kept or removed on next versions. That would be great to have an insight so that I can make the decision to use it or not. 

TIA

**Comment by triskweline** [comment, marked as answer]
We're keeping it! I'm going to set it to stable with the next release.

**Reply by alexandremjacques** [reply]
Great news! Thanks!!
